CHANGELOG.md

```diff
@@ -87,6 +87,26 @@ Main (unreleased)
 
 - Agent Management: Introduces backpressure mechanism for remote config fetching (obeys 429 request
   `Retry-After` header). (@spartan0x117)
+  
```

Comment:
Sorry, I meant to merge my other PR before you opened this, can you rebase and combine the lists?

Reply:
no problem I anticipated the merge would be necessary

Reply:
hang tight, going to merge them git was too smart auto merging

Reply:
much better